### ISSUE TITLE
Exception with non-ASCII characters in library names

### ISSUE THREAD

**Issue by shadone** [trakt/Plex-Trakt-Scrobbler#155]
Here is the exception from the logs:
```
2015-01-01 20:25:17,491 (112b8a000) :  CRITICAL (objects:73) - Exception setting attribute 'title' of object <Framework.modelling.objects.DirectoryObject object at 0x11248a810> to Push "Русские мультики" to trakt (type: <type 'str'>) (most recent call last):
  File "/Users/media/Library/Application Support/Plex Media Server/Plug-ins/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/modelling/objects.py", line 71, in _set_attribute
    el.set(convert_name(name), value)
  File "lxml.etree.pyx", line 699, in lxml.etree._Element.set (src/lxml/lxml.etree.c:34531)
  File "apihelpers.pxi", line 563, in lxml.etree._setAttributeValue (src/lxml/lxml.etree.c:15781)
  File "apihelpers.pxi", line 1366, in lxml.etree._utf8 (src/lxml/lxml.etree.c:22211)
ValueError: All strings must be XML compatible: Unicode or ASCII, no NULL bytes or control characters
```

**Comment by fuzeman** [comment]
I'll try look into this issue sometime this week.

**Comment by fuzeman** [comment]
This should be fixed in **v0.9.1.6-beta**: https://forums.plex.tv/index.php/topic/102818-rel-trakt/page-27#entry818966

Please let me know if you are still seeing this issue.

**Comment by shadone** [comment]
Indeed, v0.9.1.6 fixed this issue, thanks!
